packages/yourloops/test/integration/pages/signup-stepper/caregiver-signup.spec.tsx

```diff
@@ -56,28 +54,27 @@ describe('Signup stepper as caregiver', () => {
   })
 
   it('should be able to create a caregiver account', async () => {
-    renderPageFromHistory(history)
-    expect(history.location.pathname).toEqual('/complete-signup')
+    renderPage('/')
```

Comment:
useless redirect check I guess ? Otherwise the test should failed because not on the right page ?

Reply:
So I think that the redirect check is good, but now I am not able to test them. I tried many thing but could not figure it out. But you are right, if the redirect was wrong, the test would fail